plugins/woocommerce/includes/data-stores/class-wc-product-data-store-cpt.php

```diff
@@ -981,29 +981,22 @@ public function get_featured_product_ids() {
 	 * Check if product sku is found for any other product IDs.
 	 *
 	 * @since 3.0.0
-	 * @param int    $product_id Product ID.
 	 * @param string $sku Will be slashed to work around https://core.trac.wordpress.org/ticket/27421.
 	 * @return bool
 	 */
-	public function is_existing_sku( $product_id, $sku ) {
+	public function is_existing_sku( $sku ) {
```

Comment:
Removing `$product_id` from the method signature seems like it would break backcompat, since it is public. Two alternatives I can think of off the top:
* Rather than removing, we could change the name of the param to `$deprecated` and just not use it in the method
* Create a new method, and then deprecate this one and have it call the new method